spec/features/users/recovery_code_spec.rb

```diff
@@ -1,6 +1,8 @@
 require 'rails_helper'
 
 feature 'View recovery code' do
+  include XPathHelper
```

Comment:
Would it make more sense to add this include in `shared_examples_for_recovery_codes.rb` instead of this file and flow_spec?